portfolio/src/main/webapp/script.js

```diff
@@ -12,83 +12,122 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/* This code script executes when the website is first loaded in  */
-
-/* JavaScript "clicks" on the default tab so that content is shown when first going into the website */
-
-
-/*global variable that gives me which slide I'm currently on in the SlideShow*/
-var slideIndex;
-
-/* Performs necessary functions when page is first loaded */
-window.onload = function( ){
-    displayDefaultSection();
-    slideIndex = 1;
-    showSlides(slideIndex);
+/** 
+ * Global variable that gives me which 
+ * slide I'm currently on in the SlideShow. 
+ */
+let slideIndex = 1;
+
+/** Performs necessary functions when page is first loaded. */
+window.onload = () => {
+  displayDefaultSection();
+  showSlide(slideIndex);
 }
 
-/* Sets the default-tab by "clicking" on the "Background" tab, which has id="default-tab*/
-function displayDefaultSection(){
-    document.getElementById("default-tab").click();
+/**  
+ * Sets the default-tab by "clicking" on the "Background" tab, 
+ * which has id="default-tab.
+ */
+displayDefaultSection = () => {
+  document.getElementById("default-tab").click();
 }
 
-/* Displays certain Sections will when a specific tab is selected at the top" */
-function displaySection(event, sectionName){
-    setToNone("section-content");
-    setToDefault("section-tab", " active");
-    /* Displays the appropriate content for the sectionName associated with the tab the user just clicked on  */
-    document.getElementById(sectionName).style.display = "block";
-
-    /* Changes the styling to active for the tab the user just clicked on */
-    event.currentTarget.className += " active";
+/**  
+ * Displays certain Sections will when a specific tab is 
+ * selected at the top". 
+ * @param {event} arg The event that triggers the website to display
```

Comment:
Here and elsewhere the formatting should be "@param {type} paramName paramDescription"